automation/default_browser_params.json

```diff
@@ -8,6 +8,7 @@
     "save_content": false,
     "callstack_instrument": false,
     "dns_instrument": false,
+    "bot_mitigation": false,
```

Comment:
```suggestion
    
```
The key already exists in line 18. Consider either moving it up here or removing this entry.

Reply:
Also please remove the "random_attributes" in like 22.

Reply:
Could please explain what do you mean by `remove the "random_attributes" in like 22.`